javascript/javascript1/week1/goodboy.js

```diff
@@ -0,0 +1,18 @@
+let dogYearOfBirth = 2016;
+let dogYearFuture = 2025;
+
+//Formulas
+const dogAge = dogYearFuture - dogYearOfBirth;
+const dogYear = dogAge * 7;
+
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears === true) {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture + "."
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogAge + " human years old in " + dogYearFuture + "."
+  );
+}
```

Comment:
I have the same comment as for your ageify.js -> make sure you are aware of the distinction between let and const. Aside from that, good job!